test/ttmlir/Dialect/TTIR/reshape/reshape_test.mlir

```diff
@@ -0,0 +1,10 @@
+// RUN: ttmlir-opt --ttir-reshape-fold %s| FileCheck %s
+#any_device_tile = #tt.operand_constraint<dram|l1|tile|any_device_tile>
+module @reshape_test attributes {mhlo.num_partitions = 1 : i32, mhlo.num_replicas = 1 : i32} {
```

Comment:
Do you need these attributes?

Reply:
copies from other tests, such as t`est/ttmlir/Dialect/TTIR/slice/slice_tests_positive.mlir` to reflect the ttir code generated from stable hlo. In addition, `ttir.reshape` requires an `operand_constraints` value.

Reply:
Correct, addressed the comment.